Sources/Extensions/Foundation/EncodableExtensions.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  EncodableExtensions.swift
+//  SwifterSwift
+//
+//  Created by Alexei Kozachenko on 10/29/2018.
+//  Copyright © 2018 SwifterSwift
+//
+
+import Foundation
+
+public extension Encodable {
```

Comment:
Hey @AlexeiGitH :))
This to me feels more like a FileManager extension then a Codable. I think Codable abstracts a representation that can be encoded/decoded and we should not add file saving(or FileManager responsibility) logic or even encoding logic as an extension to this type. To me an API that maybe will make more sense is something like:
```swift
extension FileManager {
  func createFile(at path: URL, contentsEncodable: Codable, encoder: JSONEncoder = JSONEncoder()){ ...
}
```
Let me know your thoughts about that :))

Reply:
Hi @LucianoPAlmeida,

I actually have this method as a part of FileManager extensions in my repo (I think I updated the file :D) , though when I was thinking about adding it here I decided to make it as an extension of Encodable. 

It is implemented as 
```Swift
public static func encode<T: Encodable>(_ object: T, to url: URL) throws {
       ...
}
```
I think it would be convenient having it as a method of encodable class, but I totally agree that saving to a file is something that FileManager should handle. 

I think I should return to my original idea and implement it in a FileManager.